docs/reference/index-modules/translog.asciidoc

```diff
@@ -26,8 +26,9 @@ although this is rarely needed.
 === Translog settings
 
 The data in the translog is only persisted to disk when the translog is
-++fsync++ed and committed.  In the event of hardware failure, any data written
-since the previous translog commit will be lost.
+++fsync++ed and committed.  In the event of hardware failure or operating
```

Comment:
In the event of a hard failure, an operating system crash, a JVM crash, or a shard failure, any data written since the previous translog commit will be lost.